Pixel tl
Pixel br
float32 confidence
